common/supplemental/likelySubtags.xml

```diff
@@ -1036,6 +1046,7 @@ not be patched by hand, as any changes made in that fashion may be lost.
 		<likelySubtag from="und_Ahom" to="aho_Ahom_IN"/>		<!--?‧Ahom‧?	➡ Ahom‧Ahom‧India-->
 		<likelySubtag from="und_Arab" to="ar_Arab_EG"/>		<!--?‧Arabic‧?	➡ Arabic‧Arabic‧Egypt-->
 		<likelySubtag from="und_Arab_AF" to="fa_Arab_AF"/>		<!--?‧Arabic‧Afghanistan	➡ Persian‧Arabic‧Afghanistan-->
+		<likelySubtag from="und_Arab_AZ" to="az_Arab_AZ"/>		<!--?‧Arabic‧Azerbaijan	➡ Azerbaijani‧Arabic‧Azerbaijan-->
```

Comment:
I think Roozbeh had some comments about this one, since it would be exceedingly rare to have Azerbaijan written in Arabic script these days.

Reply:
@macchiati this is a branch: i'll add a comment on the PR.

Reply:
Roozbeh's comment was about Talysch. He said az_Arab_AZ is more likely than tly_Arab_AZ.

Since there are multiple writing systems for languages used in Azerbaijan and Arabic is one of them we needed a value here. Roozbeh said az_Arab_AZ made more sense -- even if its rare.

Reply:
The AZ-related changes LGTM.